docker-entrypoint.sh

```diff
@@ -0,0 +1,26 @@
+#!/usr/bin/env bash
+
+if test -f /var/www/html/inc/conf.php; then
+  if chown www-data:www-data /var/www/html/inc/conf.php ; then
+    echo "conf.php was chown'ed properly"
+  else
+    echo "There were some errors while chown'ing conf.php"
+    exit 1
+  fi
+fi
+
+if chown -R www-data:www-data /var/www/html/import/ ; then
+  echo "import folder was chown'ed properly"
+else
+  echo "There were some errors while chown'ing the import folder"
+  exit 1
+fi
+
+if chown -R www-data:www-data /var/www/html/files/ ; then
+  echo "files folder was chown'ed properly"
+else
+  echo "There were some errors while chown'ing files folder"
+  exit 1
+fi
+
+docker-php-entrypoint apache2-foreground
```

Comment:
Kinda forgot about the writeable checks, though i had them all covered.

This file works on a clean environment:

```suggestion
#!/usr/bin/env bash
paths=(install inc/Encryption.class.php inc/load.php files templates inc files lang .)

for path in ${paths[@]}; do
  if chown www-data:www-data /var/www/html/${path} ; then
    echo "${path} was chown'ed properly"
  else
    echo "There was an error while chown'ing ${path}"
    exit 1
  fi
done

docker-php-entrypoint apache2-foreground
```